sw/device/lib/arch/device_chip.c

```diff
@@ -0,0 +1,41 @@
+// Copyright lowRISC contributors.
```

Comment:
@tjaychen This file is basically a copy of `device_fpga_cw310.c` except for `kDeviceType`. Can you take a look at the constants here and let me know if we need any updates? Thanks!

Reply:
i actually think we might not need another type at the moment.  The `DV` version is supposed to correspond most closely with silicon.  For example, all the frequencies below you'd want them to match the DV one.

Reply:
SG, dropped the first commit. Thanks @tjaychen!

Reply:
@a-will 
turns out i was wrong here.
I almost immediately ran into a scenario where real silicon would be somewhere between asic and dv. So maybe we should define something for asic after all.

The main difference at the moment is the behavior of `LOG_INFO`, but there could be others in the future.